install_debs_into_image.sh

```diff
@@ -172,6 +172,9 @@ find "$IMAGEDIR/etc/ssh" -name "ssh_host_*_key*" -delete
 # restore ld.so.preload
 mv "$IMAGEDIR/etc/ld.so.preload.bak" "$IMAGEDIR/etc/ld.so.preload"
 
+# after package raspberrypi-kernel installed, install revpi-dt-blob.dtbo as default dt-blob
+install -T "$IMAGEDIR/boot/overlays/revpi-dt-blob.dtbo" "$IMAGEDIR/boot/dt-blob.bin"
```

Comment:
Why do we need this in the install_debs_to_image script?

Reply:
I think it has the same reason as it in customize_image.sh

Reply:
But it is not the same use case as for the customize_image.sh. The script is only intended to add packages to an image. And modifying the dt-blob.bin is not something the script should do.

Reply:
when the kernel package gets updated, the /boot/dt-blob.dtbo should be replace with new revpi-dt-blob.dtbo, is it?

Reply:
No, as this script has no knowledge about what has changed it might override stuff it is not intended to override. The install_debs_to_image script should I guess only install debs...

Reply:
It was added because of https://github.com/RevolutionPi/imagebakery/pull/75#discussion_r945642016

Reply:
agree, hopefully the image building can be done in CI as soon as possible.